src/inference/src/dev/threading/istreams_executor.cpp

```diff
@@ -499,5 +501,42 @@ IStreamsExecutor::Config IStreamsExecutor::Config::make_default_multi_threaded(c
     return streamExecutorConfig;
 }
 
+IStreamsExecutor::Config IStreamsExecutor::Config::reserve_cpu_threads(const IStreamsExecutor::Config& initial) {
+    auto config = initial;
+    std::vector<int> stream_ids;
+    std::string log = "[ threading ] reserve_cpu_threads " + config._name;
+    std::vector<std::string> core_type_str = {" Any core: ", " PCore: ", " ECore: ", " Logical core: "};
+    int status = config._name.find("GPU") != std::string::npos ? PLUGIN_USED : NOT_USED;
```

Comment:
Reserving CPU resource would not only use by GPU. 

Please filter "CPU" first. Then check ThreadBindingType to decide if HW plugin request reserving CPU resource